test/src/658-tarball_remove_directory/main

```diff
@@ -0,0 +1,144 @@
+
```

Comment:
This can again be a test simpler tarball.  I'd suggest one file and one nested sub directory to test the recursive delete.